Solutions/CohesitySecurity/Tests/az.py

```diff
@@ -155,11 +156,49 @@ def search_alert_id_in_incident(alert_id, resource_group, workspace_name):
     return json.loads(result.stdout) if json.loads(result.stdout) else None
 
 
+def get_latest_playbook_run(
+    access_token, subscription_id, resource_group, playbook_name
+):
+    headers = {
+        "Authorization": "Bearer " + access_token,
+        "Content-Type": "application/json",
+    }
+    url = (
+        "https://management.azure.com/subscriptions/{}/resourceGroups/{}/"
+        "providers/Microsoft.Logic/workflows/{}/runs?api-version=2016-06-01"
+        "&$top=1"
+    ).format(subscription_id, resource_group, playbook_name)
+    response = requests.get(url, headers=headers)
```

Comment:
Can we do some basic checks like whether the HTTP request succeeded or not?

Reply:
done